app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryState.js

```diff
@@ -4,12 +4,18 @@ import * as Story from '../../../models/beta/story';
 import { editingStoryPropTypesShape, isUnestimatedFeature } from '../../../models/beta/story';
 import ExpandedStorySection from './ExpandedStorySection';
 
+const handleState = (story) => {
+  if (story._editing.state === "unscheduled" && typeof story._editing.estimate === "number") return "unstarted";
```

Comment:
It would be better if these business rules were inside the models (https://github.com/Codeminer42/cm42-central/blob/master/app/assets/javascripts/models/beta/story.js#L134), which are used by the reducers (https://github.com/Codeminer42/cm42-central/blob/master/app/assets/javascripts/reducers/stories.js#L40).

This way we have all business rules (like "an unscheduled story is always unstarded") in a single place. What do you think?

Reply:
Sure, I will move then.